daemon/commands.c

```diff
@@ -169,6 +173,23 @@ static cmd_status_t cmd_fatal_execute(char *args, char **message)
     return CMD_STATUS_SUCCESS;
 }
 
+static cmd_status_t cmd_reload_labels_execute(char *args, char **message)
+{
+    (void)args;
+    (void)message;
+    info("COMMAND: reloading host labels.");
+    reload_host_labels();
+    struct label *l=localhost->labels;
+    BUFFER *wb = buffer_create(10);
+    while (l != NULL) {
+        buffer_sprintf(wb,"Label [source id=%s]: \"%s\" -> \"%s\"\n", translate_label_source(l->label_source), l->key, l->value);
+        l = l->next;
+    }
```

Comment:
I think we need `netdata_rwlock_rdlock(&host->labels_rwlock)` for this loop.

Reply:
Correct. I'll add one.

Reply:
I agree, I brought the missing lock/unlock for this code.